net/dns/manager_windows.go

```diff
@@ -34,6 +36,8 @@ const (
 	versionKey = `SOFTWARE\Microsoft\Windows NT\CurrentVersion`
 )
 
+var configureWSL, _ = strconv.ParseBool(os.Getenv("TS_DEBUG_CONFIGURE_WSL"))
```

Comment:
We'd advise people to set a System Environment variable? It looks like those would apply to the Windows service tailscaled.